convert2rhel/unit_tests/checks_test.py

```diff
@@ -1215,3 +1218,43 @@ def test_is_loaded_kernel_latest_unsupported_skip(
         is_loaded_kernel_latest()
 
     assert expected in caplog.records[-1].message
+
+
+@pytest.mark.parametrize(
+    ("no_rhsm", "dbus_running", "log_msg"),
+    (
+        (True, True, "Skipping the check because we have been asked not to subscribe this system to rhsm."),
+        (True, False, "Skipping the check because we have been asked not to subscribe this system to rhsm."),
```

Comment:
```suggestion
        (True, True, "Skipping the check because we have been asked not to subscribe this system to RHSM."),
        (True, False, "Skipping the check because we have been asked not to subscribe this system to RHSM."),
```